e2e/app/page/create-account-page.ts

```diff
@@ -37,8 +38,8 @@ export const EDUCATION_LEVEL_VALUE = {
 };
 
 export const LABEL_ALIAS = {
-  READ_UNDERSTAND_PRIVACY_STATEMENT: 'I have read, understand, and agree to the All of Us Program Privacy Statement.',
-  READ_UNDERSTAND_TERMS_OF_USE: 'I have read, understand, and agree to the Terms of Use described above.',
+  READ_PRIVACY_STATEMENT: 'I have read, understand, and agree to the All of Us Program Privacy Statement',
+  READ_TERMS_OF_USE: 'I have read, understand, and agree to the Terms of Use described above',
```

Comment:
lint check: Had to shorten the label alias in orde to keep length of line under 140 characters. Also for some other long alias.

Reply:
You could always break the string over two lines.